whois/__init__.py

```diff
@@ -1,20 +1,21 @@
 # -*- coding: utf-8 -*-
+"""Library for querying and parsing results from whois databases."""
 
 from __future__ import print_function
 from __future__ import absolute_import
 from __future__ import unicode_literals
 from __future__ import division
 from future import standard_library
 standard_library.install_aliases()
-from builtins import *
-import re
-import sys
-import os
-import subprocess
-import socket
-from .parser import WhoisEntry
-from .whois import NICClient
-import logging
+from builtins import *  # noqa
+import re  # noqa: E402
```

Comment:
why are these lint comments needed here?

Reply:
wow I've been asleep for so long.. should've followed up here but I suddenly didn't have enough energy time and focus.

for those lines specifically, I get those errors:

~~~
whois/__init__.py:9:1: F403 'from builtins import *' used; unable to detect undefined names
whois/__init__.py:9:1: F401 'builtins.*' imported but unused
whois/__init__.py:9:1: E402 module level import not at top of file
~~~

flake8 usually complains about `import *` but maybe we can disable more specific errors for the first line. in the other comment about the `import *` line, do you mean that we should rather explicitly import the parts of "builtins" that are needed instead of using "*" ?

for the second line, I guess it's complaining because on line 8 there's a call to `standard_library.install_aliases()`, so something gets executed before other imports. I've just tested moving that call down below all of the other import lines and the library still seems to be working properly while the flake8 issue is gone.
do you remember off-hand if there was a reason for the call to be placed above the other imports? by tracing the file with git I get to the commit "import from mercurial" so I can't see why the line was placed there.
if moving the call does not seem to be problematic, then we can do this and remove all of the "noqa: E402" comments.